app/models/canonical/book.rb

```diff
@@ -49,6 +54,10 @@ def self.unique_identifier
       :item_code
     end
 
+    def recipe?
```

Comment:
Add this method to make a somewhat uniform API between possible associations to the `RecipesCanonicalIngredient` model.